airflow/api_connexion/openapi/v1.yaml

```diff
@@ -412,25 +412,38 @@ paths:
         - $ref: '#/components/parameters/PageOffset'
         - $ref: '#/components/parameters/OrderBy'
         - $ref: '#/components/parameters/FilterTags'
-        - name: only_active
-          in: query
-          schema:
-            type: boolean
-            default: true
-          required: false
-          description: |
-            Only return active DAGs.
-
-            *New in version 2.1.1*
-        - name: dag_id_pattern
-          in: query
-          schema:
-            type: string
-          required: false
-          description: |
-            If set, only return DAGs with dag_ids matching this pattern.
+        - $ref: '#/components/parameters/OnlyActive'
+        - $ref: '#/components/parameters/DagIdPattern'
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DAGCollection'
+        '401':
+          $ref: '#/components/responses/Unauthenticated'
 
-            *New in version 2.3.0*
+    patch:
+      summary: Update DAGs
+      x-openapi-router-controller: airflow.api_connexion.endpoints.dag_endpoint
+      operationId: patch_dags
+      parameters:
+        - $ref: '#/components/parameters/PageLimit'
+        - $ref: '#/components/parameters/PageOffset'
+        - $ref: '#/components/parameters/FilterTags'
+        - $ref: '#/components/parameters/UpdateMask'
+        - $ref: '#/components/parameters/OnlyActive'
+        - $ref: '#/components/parameters/DagIdPattern'
```

Comment:
This endpoint by default operates on _all_ DAGS by default right? That feels like a bit of a foot gun, and I _think_ I'd like to make dag id pattern required here, i.e. `?dag_id_pattern=%` is required to operate on all dags.

WDYT?

Reply:
Yes, I agree, I can add that fix in this PR. WDYT about setting `dag_id_pattern` as `~` to return all dags if `dag_id_pattern` is required?

Reply:
So long as it's documented, sounds good

Reply:
This only applies to the bulk patch endpoint, others can stay as they are

Reply:
Does that mean that the bulk `get` endpoint should continue to treat `dag_pattern_id` as optional and default to returning all dags if no pattern is provided?

Reply:
Yeah, the behaviour of bulk `get` shouldn't change.